Source/SwiftLintFramework/Rules/Lint/MarkRule.swift

```diff
@@ -183,17 +183,25 @@ public struct MarkRule: CorrectableRule, ConfigurationProviderRule {
     }
 
     private func violationRanges(in file: File, matching pattern: String) -> [NSRange] {
+        // find all comment Kinds first and then run matching patter over each of those
```

Comment:
Just a small typo: `patter` should be `pattern`

Reply:
👍 fixed